### ISSUE TITLE
Angular 2 component

### ISSUE THREAD

**Issue by bill-long** [fraywing/textAngular#971]
Any work happening on an Angular 2 component version of textAngular? I know it's early yet. I've just been working on a proof-of-concept where this would be useful.

**Comment by kesarion** [comment]
An Angular 2 version would be fantastic now with the beta coming to a close. I think we'll start to see many components get an update.

**Comment by pbassut** [comment]
At this point we're focusing on getting the library stable and bug-free. This is more important than any enhancement.
But, if you would like to discuss this and submit a PR, I would love to!

**Comment by jprystowsky** [comment]
Any update on Angular 2 support?

**Comment by glinkot** [comment]
Yes this editor looks amazing but Angular (2+) support would be needed before I could try it :)

**Comment by nkmdev** [comment]
Any work happening on an Angular(5) component version of textAngular?

**Comment by frostme** [comment]
#1590

**Comment by frostme** [comment]
is this library maintained anymore? if so, is there an update on the angular 2+ support? or does anyone know of a good replacement for angular 2+?

**Comment by akvaliya** [comment]
Any updates on angular 2+ support?
